packages/main/src/Dialog.js

```diff
@@ -212,10 +215,43 @@ class Dialog extends Popup {
 	}
 
 	onExitDOM() {
```

Comment:
is it really needed to delete the handlers? Can't we let them be and initialize them in the constructor()?